bridge-to-the-moon/src/challengeClaim.js

```diff
@@ -0,0 +1,126 @@
+const StateMachine = require('javascript-state-machine')
+const VerificationGame = require('./verificationGames/challenger')
+const BlockEmitter = require('./utils/blockemitter')
+const waitForEvent = require('./utils/waitForEvent')
+const timeout = require('./utils/timeout')
+
+module.exports = (web3, api) => ({
+  run: async (cmd, claim, autoDeposit = false) => new Promise(async (resolve, reject) => {
+    try {
+      const { claimManager } = api
+      const me = web3.eth.defaultAccount
```

Comment:
This should be configurable.

Reply:
Created an issue for this: https://github.com/TrueBitFoundation/scrypt-interactive/issues/38.